README.md

```diff
@@ -24,3 +24,19 @@ Références :
 - [construire l'url](https://developer.matomo.org/guides/reporting-api-tutorial#build-the-url)
 - [appeler la méthode](Live.getLastVisitsDetails)
 - [l'idSite à utiliser](https://developer.matomo.org/api-reference/reporting-api)
+
+La procédure d'installation pour convertir un notebook en script (grace à `jupyter nbconvert`)
+```
+sudo apt-get install python3
+sudo apt-get install ipython
+sudo apt-get install jupyter
+jupyter nbconvert --to script 1.fetch_logs.ipynb
+virtualenv -p /usr/bin/python3 py3env
+source ./py3env/bin/activate
+pip install requests
```

Comment:
You probably can install `jupyter` via `pip` too, you only need `python3` though `apt`.